include/myst/thread.h

```diff
@@ -187,6 +187,9 @@ struct myst_thread
     myst_spinlock_t* thread_lock;
     struct myst_thread* group_prev;
     struct myst_thread* group_next;
+
+    /* thread name */
+    char name[16];
```

Comment:
Is this what Linux expected?

Reply:
Yes. 

https://elixir.bootlin.com/linux/latest/source/include/linux/sched.h#L957
https://elixir.bootlin.com/linux/latest/source/include/linux/sched.h#L215